src/features/navigation/RootNavigator/RootNavigator.tsx

```diff
@@ -50,7 +52,7 @@ export const RootNavigator: React.FC = () => {
       )}
       {/* The components below are those for which we do not want
       their rendering to happen while the splash is displayed. */}
-      {isSplashScreenHidden && <PrivacyPolicy navigationRef={navigationRef} />}
+      {isSplashScreenHidden && consentAsked && <PrivacyPolicy navigationRef={navigationRef} />}
```

Comment:
L'objectif est de cacher cette modale sur iOS non ?

Reply:
Alors d'après ce que j'ai compris en lisant les docs que j'ai mis en références, `PrivacyPolicy` concerne exclusivement Firebase (données de navigation pour générer des statistiques...). La modale ATT est nécessaire pour qu'AppsFlyer ait accès à l'IDFA de l'utilisateur, s'il l'autorise, sinon on peut toujours utiliser AppsFlyer mais on aura pas accès à toutes les données pour tracker l'utilisateur entre différents sites (qui est le but d'ATT:  `Request user authorization to access app-related data for tracking the user or the device.`)

Reply:
> Starting with iOS 14.5, iPadOS 14.5, and tvOS 14.5, you’ll need to receive the user’s permission through the AppTrackingTransparency framework to track them or access their device’s advertising identifier. Tracking refers to the act of linking user or device data collected from your app with user or device data collected from other companies’ apps, websites, or offline properties for targeted advertising or advertising measurement purposes. Tracking also refers to sharing user or device data with data brokers.

Reply:
Dans notre cas, avec AppsFlyer, on utilise pour ce cas:
> Placing a third-party SDK in your app that combines user data from your app with user data from other developers’ apps to target advertising or measure advertising efficiency, even if you don’t use the SDK for these purposes. For example, using an analytics SDK that repurposes the data it collects from your app to enable targeted advertising in other developers’ apps.

[source](https://developer.apple.com/app-store/user-privacy-and-data-use/)

Reply:
D'ailleurs pour totalement découpler les 2 types de tracker, il faudrait lancer firebase sans qu'il récupère l'IDFA.

C'est pour ça que j'ai upgradé firebase à 1.14.1, pour qu'il supporte `RNFirebaseAnalyticsWithoutAdIdSupport`

Doc firebase: https://rnfirebase.io/analytics/usage#disable-ad-id-usage-on-ios

Reply:
Donc on ne veut pas faire les deux trackers (Firebase + Appsflyers) via la même modale ATT ? 
Ca fait pas beaucoup de modales à l'arrivée sur l'app ? 
1. ATT
2. Notifs batch
3. Politique de confidentialité
4. On enchaine sur les tutoriels...

Reply:
À valider avec les POs of course

Reply:
oui je suis d'accord que ça fait bcp, mais qu'est-ce qu'on peut faire?
On peut controler l'ordre des pop-ups (ce que j'ai fait). On peut afficher la pop up de notifiication plus tard (pas fait mais faisable)